Sources/ComposableArchitecture/Documentation.docc/Extensions/Deprecations/EffectDeprecations.md

```diff
@@ -10,41 +10,41 @@ Avoid using deprecated APIs in your app. Select a method to see the replacement
 
 ### Creating an effect
 
-- ``Effect/task(priority:operation:)``
+- ``EffectPublisher/task(priority:operation:)``
 
 ### Cancellation
 
-- ``Effect/cancel(ids:)-9tnmm``
+- ``EffectPublisher/cancel(ids:)-9tnmm``
```

Comment:
- [ ] Just a note that @mbrandonw and I should audit these after merging to make sure the identifiers haven't "drifted".

Reply:
I think it can't resolve on my machine, but I'm not exactly sure how it works, as Xcode doesn't suggest anything. Because they're deprecated, right?

Reply:
DocC's Xcode integration is sometimes a bit flakey. We've found that running the `swift package` command to spin up a server helps:

```sh
swift package --disable-sandbox preview-documentation --target ComposableArchitecture
```

And then we've been manually navigating to the method and copying its identifier when it drifts.